.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,6 @@
 
 ##### Contributor checklist
 
-- [ ] Provided the tests for the changes.
-- [ ] Gave a clear one-line description in the PR (that the maintainers can add to CHANGELOG.md on release).
-- [ ] Assign the PR to an existing or new milestone for the target version (following [Semantic Versioning](https://blog.versioneye.com/2014/01/16/semantic-versioning/)).
+- \[ \] Provided the tests for the changes.
```

Comment:
Hm, this change might be wrong

Reply:
Hi @atugushev ! Mdformat author here.

Mdformat is CommonMark compliant by default, and CommonMark does not know about the task list syntax, this is why this happens. You can install [`mdformat-gfm`](https://github.com/hukkinj1/mdformat-gfm) to change the parser's syntax specification from CommonMark to GFM. :smile: 

In a pre-commit setup this can be done by adding `additional_dependencies: [mdformat-gfm]` to `.pre-commit-config.yaml`.